src/RemoteInboxNotifications/RemoteInboxNotificationsEngine.php

```diff
@@ -26,14 +26,6 @@ class RemoteInboxNotificationsEngine {
 	public static function init() {
 		// Continue init via admin_init.
 		add_action( 'admin_init', array( __CLASS__, 'on_admin_init' ) );
-
```

Comment:
Part of me thinks we leave this action in place here, but instead don't call `self::run` in the `update_profile_option` method. Or we add logic to the `run` class to not have it call out to the remote endpoint.